code/modules/projectiles/projectile.dm

```diff
@@ -667,17 +667,23 @@
 			playsound(target_mob, "bullet_miss", 75, 1)
 		return FALSE
 
+	var/impacted_organ = parse_zone(def_zone)
+	if(istype(target_mob, /mob/living/simple_mob))
+		var/mob/living/simple_mob/SM = target_mob
+		var/decl/mob_organ_names/organ_plan = GET_DECL(SM.organ_names)

```

Comment:
Should the mob not just replace the path with this reference on initialize? That is, every time you want to ask what's in it, you don't have to fetch it out of the decl repo, thus saving some small overhead?